server.go

```diff
@@ -34,17 +34,19 @@ func verifyToken(bot *Bot, w http.ResponseWriter, r *http.Request) {
 }
 
 type receive struct {
+	Object  string  `json:"object"`
```

Comment:
At least we have to set it to  json:"-" I think.
2016年5月18日 20:23於 "frodsan" <notifications@github.com>寫道：

> In server.go <https://github.com/frodsan/fbot/pull/1#discussion_r63692587>
> :
>
> > @@ -34,17 +34,19 @@ func verifyToken(bot *Bot, w http.ResponseWriter, r *http.Request) {
> >  }
> >
> >  type receive struct {
> > +	Object  string  `json:"object"`
>
> Object is a string and always "page". Not sure if we need this:
>
>
> https://developers.facebook.com/docs/messenger-platform/webhook-reference#common_format
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/frodsan/fbot/pull/1/files/eeb7ca1f0fb47cd09ae101dd9707bdee44a33268#r63692587>
>